File: leetcode2/1easy/이총명/Q1619.java
class Solution {
    public double trimMean(int[] arr) {
        Arrays.sort(arr);
        int percent5 = arr.length/20;
        double sum = 0.0;
        int cnt = 0;
        for(int i = 0+percent5; i < arr.length - percent5; i++){
            sum += arr[i];
            cnt++;
        }
        return sum/cnt;
    }
}

File: leetcode2/1easy/강병호/Q1619.py
class Solution:
    def trimMean(self, arr: List[int]) -> float:
        arr.sort()

        n = len(arr)
        trimNumCnt = int(n * 0.05)

        trimedArr = arr[trimNumCnt:n-trimNumCnt]
        print(trimedArr)

        trimedMean = sum(trimedArr) / (n- 2*trimNumCnt)

        print(trimedMean)

        return round(trimedMean, 5)

